yarn.lock

```diff
@@ -1815,6 +1817,34 @@ esbuild@^0.19.2, esbuild@^0.19.3:
     "@esbuild/win32-ia32" "0.19.9"
     "@esbuild/win32-x64" "0.19.9"
 
+esbuild@~0.18.20:
```

Comment:
there is a duplication of esbuild, but I think it will be upgraded soon. The package looks well maintained.